tasks/eth/base-002-fp-upgrade/.env

```diff
@@ -0,0 +1,8 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0x95703e0982140d16f8eba6d158fccede42f04a4c
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+# This is Base's safe but the Nested.just file uses council as a keyword and
+# we want to minimize changes
+COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1
```

Comment:
Not sure to understand for now, why we keeping `COUNCIL_SAFE` seems related because of the `Nested.just` but we should find another alternative in the future to make this clearer to avoid confusion imo. 
``` 
# This is Base's safe but the Nested.just file uses council as a keyword and
# we want to minimize changes
COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1
```

Reply:
Yeah I agree, perhaps it could be signer1, signer2, and signer3 (for Base and Unichain's 3/3). I think a standard naming convention would be good as we move towards a Superchain focus.